.kitchen.yml

```diff
@@ -21,10 +22,10 @@ platforms:
   driver_config:
     box: opscode-centos-6.5
     box_url: http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_centos-6.5_chef-provisionerless.box
-- name: oracle-6.4
```

Comment:
because I want to have a clean run on `kitchen test -c` and the oracle box is preventing this (a repo error). the CI test-matrix can include oracle 6.4. I just do not want to have an error in the default settings